paparazzi/src/main/java/app/cash/paparazzi/HtmlReportWriter.kt

```diff
@@ -121,6 +121,10 @@ public class HtmlReportWriter @JvmOverloads constructor(
   private fun hash(image: BufferedImage): String {
     val hashingSink = HashingSink.sha1(blackholeSink())
     hashingSink.buffer().use { sink ->
+      // write image dimensions to avoid views with the same contents but different dimensions
+      // causing a hash collision.
+      sink.writeInt(image.width)
+      sink.writeInt(image.height)
```

Comment:
As mentioned in: https://github.com/cashapp/paparazzi/issues/1527#issuecomment-2278496452, I think this will only solve a subset of the root cause problem.  How about instead writing the (x,y) values prior to the pixel rgb data?

Reply:
Small note: https://github.com/cashapp/paparazzi/issues/1527#issuecomment-2279741497

Base on the upper: writing (x,y) prior the pixel rgb data will solve the collision problem but will not give extra protection (unless there are other edge cases) while increasing the hashing time a bit (insignificantly though)